modules/importer/src/runner/common/walker/git.rs

```diff
@@ -202,7 +202,8 @@ where
                 builder.branch(branch);
             }
 
-            let fo = Self::create_fetch_options();
+            let mut fo = Self::create_fetch_options();
+            fo.depth(1);
```

Comment:
This `depth` is depth of git history fetched, yeah? Not directory tree depth? If so, LGTM.

Reply:
I believe so, yes. I worry a bit that subsequent updates might somehow be affected by the initial shallow clone, but I couldn't find any evidence of that by bouncing the process. Seemed to pick up where it left off each time.